lib/modules/manager/tekton/extract.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { Fixtures } from '../../../../test/fixtures';
+import { extractPackageFile } from '.';
+
+describe('modules/manager/tekton/extract', () => {
+  describe('extractPackageFile()', () => {
+    it('extracts deps from a file', () => {
+      expect(
+        extractPackageFile(Fixtures.get('multi-doc.yaml'), 'test-file.yaml')
+      ).toMatchSnapshot();
```

Comment:
add assertion for deps length

Reply:
The snapshot performs an exact match, which covers this and more. Wouldn't that be a redundant check?

Reply:
no, snapshots tend to be silently updated with wrong changes

Reply:
Added length check.